runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -182,8 +182,8 @@ typedef enum
 
 typedef enum
 {
-  /** Invalid */
-  NNFW_TRAIN_LOSS_REDUCTION_INVALID = 0,
+  /** Auto */
+  NNFW_TRAIN_LOSS_REDUCTION_AUTO = 0,
```

Comment:
How about keep both `invalid` and `auto` here? 

Since a api for getting train_info - `nnfw_train_get_traininfo` will be added, There `invalid` is also necessary in API space  to represent(`LossReductionType::Auto` in core).

Reply:
Otherwise, looks good to me. 👍

Reply:
How about adding `Invalid` enum to both `Loss` and `Optimizer` at once? :)

Reply:
> How about adding Invalid enum to both Loss and Optimizer at the same time? 

Oh I see. No problem. :smile: 
If so, I could add `invalid` in same time for both `core/ir/..` and `api/../nnfw_experimental`, when it become necessary.